src/bowlingGame.js

```diff
@@ -0,0 +1,32 @@
+class BowlingGame {
+  constructor() {
+  this.rolls = [];
+  this.total =[];
+  }
+
+  roll(pins){
+    this.rolls.push(pins)
+  }
+
+  score() {
```

Comment:
I had quite a similar approach here but wondering if this one could be split into different functions